mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -185,6 +186,43 @@ void AbstractControllerExecution::setVelocityCmd(const geometry_msgs::TwistStamp
   // TODO so there should be no loss of information in the feedback stream
 }
 
+bool AbstractControllerExecution::checkVelocityIgnore(const geometry_msgs::Twist& cmd_velocity)
+{
+  bool ignored_exceed_tolerance = false;
+
+  bool robot_stopped = robot_info_.isRobotStopped(1e-3, 1e-3);
```

Comment:
```suggestion
  const bool robot_stopped = robot_info_.isRobotStopped(1e-3, 1e-3);
```
S: use const whenever you don't change the value after